test/README.md

```diff
@@ -335,6 +383,16 @@ images that use RPMs created from source (not already published releases).
 ./test/bin/build_images.sh -s
 ```
 
+##### Creating BootC Images
+
+Use `./test/bin/build_bootc_images.sh` to build all of the images for all of the
```

Comment:
This does not work for the script. It displays a usage message if run without arguments.